src/demo/helpers.js

```diff
@@ -137,3 +144,28 @@ export const generateColorPalette = (colors, bodyIndex = null) => {
     knnData: bodyColor.knnData
   };
 };
+
+export const currentRunTime = state => {
```

Comment:
Some comments for these new functions would be good.

Reply:
good call, adding those now